### PR TITLE
Add ukiyo

### PR DESCRIPTION
**Pull request by CyberFlameGO**
Adds Ukiyo

### CODE REVIEW

**Comment by AdamAtomus** [comment]
my guy i added ukiyo already with the same id...

File: servers.json [hunk 1]
```diff
@@ -6,6 +6,13 @@
             "edenmine.eu"
         ]
     },
+    {
+        "id": "ukiyo",
```
Comment (GrowlyX) [inline servers.json:10]: Is `ukiyo.rip` and `ukiyomc.com` (this pr) the same server?

If they are, which one would you like to keep, as the entries are conflicting:

image

Comment (KuroWolf) [reply servers.json:10]: Different servers.

Comment (CyberFlameGO) [reply servers.json:10]: Hey there @GrowlyX, these are different servers

Comment (GrowlyX) [reply servers.json:10]: > Hey there @GrowlyX, these are different servers

Is it okay if I change the id of your server to `ukiyomc`, so there are no conflicts with the other server?

**Review by GrowlyX — APPROVED** [review]

File: servers.json (hunk 1 above)
Comment (CyberFlameGO) [reply servers.json]: Sure, though when I created the pull requests I saw no upstream conflicts

**Pull request merged**